e2e/tests/content-manager/editview.spec.ts

```diff
@@ -0,0 +1,179 @@
+import { test, expect } from '@playwright/test';
+import { login } from '../../utils/login';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+
+test.describe('Edit View', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath('./e2e/data/with-admin.tar');
+    await page.goto('/admin');
+    await login({ page });
+  });
+
+  test.describe('Collection Type', () => {
+    test('A user should be able to navigate to the EditView of the content manager to create, save, publish, unpublish & delete a new entry', async ({
+      page,
+    }) => {
+      await page.getByRole('link', { name: 'Content Manager' }).click();
+      await page.getByRole('link', { name: /Create new entry/ }).click();
+
+      /**
```

Comment:
You could explain the next step in this comment
```
/**
* Now we're in the edit view and we try to create a new article and save.
*/
```

Reply:
I did not work in this file so I won't make any updates there. Sorry for the confusion I tried to outline the files I modified in the PR description